### DISCUSSION TITLE
How could I tell piku to re-generate certificates for one app?

### DISCUSSION THREAD

**Discussion by mariusa** [piku/piku#298] [Q&A] [answered]
I didn't have acme.sh installed when an app was created. Now it's installed.
How could I tell piku to re-generate certificates for an app?

Thanks

**Comment by mariusa** [comment]
piku restart does it :)

Got stuck though on
Verify error:IPV6 ip: Invalid response from http://app.com/.well-known/acme-challenge/... : 403
I don't see anything relevant in acme.sh log, not sure why it's forbidden since nginx/app.conf has 
```
 location ^~ /.well-known/acme-challenge {
    allow all;
    root /home/piku/.piku/acme;
  }
```

**Comment by chr15m** [comment]
Sometimes this kind of error occurs if the domain you are using has not yet propagated through DNS in such a way that the Let's Encrypt servers can resolve them to the correct IP. The solution there is to wait.

If it's not that you will have to dig in yourself to try and figure out why you're seeing that on your setup.

**Reply by mariusa** [reply, marked as answer]
/home/piku must be `chgrp nginx` and mode `0710` for nginx to be able to read acme challenge
